packages/app-mobile/components/screens/search.tsx

```diff
@@ -188,8 +188,13 @@ class SearchScreenComponent extends BaseScreenComponent {
 							value={this.state.query}
 							selectionColor={theme.textSelectionColor}
 							keyboardAppearance={theme.keyboardAppearance}
+							accessibilityHint={_('Search for...')}
+							returnKeyType='search'
 						/>
-						<TouchableHighlight onPress={() => this.clearButton_press()}>
+						<TouchableHighlight
+							onPress={() => this.clearButton_press()}
+							accessibilityLabel={_('Reset search input')}
```

Comment:
Would it make sense to name this "Clear" (for which we already have a translation)?